text_test.go

```diff
@@ -228,3 +229,78 @@ func TestTextFatal(t *testing.T) {
 	}
 	t.Fatalf("process ran with err %v, want exit status 1", err)
 }
+
+func TestTextValueStyles(t *testing.T) {
+	var buf bytes.Buffer
+	logger := New(WithOutput(&buf)).(*logger)
+	logger.noStyles = false
+	ValueStyle = lipgloss.NewStyle().Bold(true)
+	cases := []struct {
+		name     string
+		expected string
+		msg      string
+		kvs      []interface{}
+		f        func(msg interface{}, kvs ...interface{})
+	}{
```

Comment:
Could you add a test case for a multiline value?

Reply:
Just pushed up that, including having to update `writeIdent` to account for styling the values after escaping as well. I might look into extracting the post-styling after escaping into the escape function itself.